ext/hosting.py

```diff
@@ -43,6 +43,13 @@ async def host(self, ctx, mode, notification_type, *, gamemode='Not Defined'):
         else:
             return await ctx.send('That is an invalid mode.', delete_after=5)
 
+        if mode.lower() == "coven":
+            if gamemode.lower != "classic" or "ranked practice" or "rankedpractice" or "ranked practise" or "rankedpractise" or "mafia returns" or "mafiareturns" or "custom" or "all any" or "allany" or "any all" or "anyall" "vip" or "lovers" or "town traitor" or "towntraitor" or "rotating gamemode" or "rotatinggamemode":
```

Comment:
That's not how the or operator works in python. 
That always evaluates to True.

Reply:
Yeh I forgot to push my new changes

Reply:
I already fixed.